fastlane/Fastfile

```diff
@@ -3,6 +3,7 @@ require_relative 'test_center_utils'
 # Retrieves the example code from each action and runs it to ensure that everything
 # is working as intended.
 lane :run_examples do
+  cocoapods(podfile: 'KiwiDemo/Podfile')
```

Comment:
This can go in the example lane for the multi_scan examples that deals with this.

Reply:
https://github.com/yousefhamza/fastlane-plugin-test_center/blob/feature/invocation_based_tests/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L335